src/mlpack/methods/logistic_regression/logistic_regression_function_impl.hpp

```diff
@@ -125,6 +125,41 @@ double LogisticRegressionFunction<MatType>::Evaluate(
     return -log(1.0 - sigmoid) + regularization;
 }
 
+/**
+ * Evaluate the logistic regression objective function given the estimated
+ * parameters for a given batch from a given point.
+ */
+template<typename MatType>
+double LogisticRegressionFunction<MatType>::Evaluate(
+                  const arma::mat& parameters,
+                  const size_t begin,
+                  const size_t batchSize) const
+{
+  // Calculating the regularization term.
+  const double regularization = 0.5 * lambda *
+    arma::dot(parameters.col(0).subvec(begin, batchSize - 1),
+              parameters.col(0).subvec(begin, batchSize - 1));
```

Comment:
Can you make sure that wrapped lines are indented 4 spaces not 2 from the previous line please?